src/plugins/vis_type_timeseries/server/lib/search_strategies/capabilities/default_search_capabilities.ts

```diff
@@ -51,7 +50,7 @@ export class DefaultSearchCapabilities {
   }
 
   public get searchTimezone() {
-    return getTimezoneFromRequest(this.request);
+    return getTimezoneFromRequest(this.request as VisTypeTimeseriesVisDataRequest);
```

Comment:
are you sure we need that casting here? in line 24 you already set `VisTypeTimeseriesRequest` type for request. `getTimezoneFromRequest` also works with `VisTypeTimeseriesVisDataRequest`

Reply:
`VisTypeTimeseriesRequest` is a union type of `VisTypeTimeseriesFieldsRequest | VisTypeTimeseriesVisDataRequest`, 
but `getTimezoneFromRequest` expects only `VisTypeTimeseriesVisDataRequest`.
If you don't like type casting here, I made a type guard in the recent commit (but it looks overloaded IMHO)